services/cases-api/lambdas/updateCase.js

```diff
@@ -9,17 +13,44 @@ import { decodeToken } from '../../../libs/token';
 import { objectWithoutProperties } from '../../../libs/objects';
 import { getStatusByType } from '../../../libs/caseStatuses';
 
+import { getCaseWhereUserIsApplicant } from '../helpers/dynamo';
+import { updateCaseValidationSchema } from '../helpers/schema';
+
 export async function main(event) {
   const requestJsonBody = JSON.parse(event.body);
   const { id } = event.pathParameters;
+  const { personalNumber } = decodeToken(event);
 
   if (!id) {
-    return response.failure(new BadRequestError('missing [id] in query path'));
+    return response.failure(new BadRequestError('Missing required path parameter "id"'));
+  }
+
+  const { error, validatedEventBody } = updateCaseValidationSchema.validate(requestJsonBody, {
+    abortEarly: false,
+  });
+  if (error) {
+    response.failure(new BadRequestError(error.message.replace(/"/g, "'")));
+  }
+
+  const {
+    provider,
+    details,
+    currentFormId,
+    currentPosition,
+    answers,
+    signature,
+  } = validatedEventBody;
+
+  const [getCaseError] = await to(getCaseWhereUserIsApplicant(id, personalNumber));
+  if (getCaseError) {
+    return response.failure(
+      new ResourceNotFoundError('The user does not have any case with the provided case id')
+    );
   }
 
-  const { provider, statusType, details, currentFormId, currentPosition, answers } =
-    requestJsonBody;
+  const newCaseStatus = getNewCaseStatus(answers, signature);
```

Comment:
`signature` is only used here; it is never added/updated to the case (I assume it should be, like the other properties). Add `if (signature) { UpdateExpression.push(...) }` etc. together with the rest further down.

Reply:
Signature in this first implementation do not require any saving towards the case-database. But @dannil76 is working on an improved signature for multiple persons, where the signature will be updated for each person in the persons attribute on a case (y)